src/components/__tests__/FontAwesomeIcon.test.js

```diff
@@ -2,9 +2,10 @@
  * @jest-environment jsdom
  */
 
-import { library } from '@fortawesome/fontawesome-svg-core'
 import { faClose, faUser } from '@fortawesome/free-solid-svg-icons'
 import { faCoffee, faCircle, faAlien, faDog } from '../__fixtures__/icons'
+import { ICON_PACKS_STARTING_VERSION, SVG_CORE_VERSION } from '../../utils'
```

Comment:
I added this import here and the "semver" import.
The other imports were previously added, I just alphabetized them.